db/deploy/create_roles.sql

```diff
@@ -10,7 +10,7 @@ begin
     select true
     from pg_catalog.pg_roles
     where rolname = 'ccbc_guest') then
-    
+
```

Comment:
It looks like we did a rework and then didn't change the file other than some spacing fixes (I'm guessing be prettier). I think we should remove the rework for readability

Reply:
I'm confused as to what you mean, we did the sqitch rework to add the `ccbc_job_executor` in `create_roles@1.5.0.sql`. This white space change is from the pre commit hook.

Reply:
Ah I misunderstood, the `.sql` file without the tag is the latest one while the tagged version is the old version, so you have them swapped in this case. Just need to change the file names in this case then. When this is deployed in prod it won't create the latest roles